# HG changeset patch
# User Rik <rik@octave.org>
# Date 1348946018 25200
#      Sat Sep 29 12:13:38 2012 -0700
# Node ID 2d7bdbdd9f5d0d7aaadce335a47a3f92f8c63013
# Parent  4d960b078272060550a98f506f6dfff300f2e808
configure.ac: Remove commented and obsolete test for Sun platform.

* configure.ac: Remove commented and obsolete test for Sun platform.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -445,28 +445,16 @@ if test "$cross_compiling" = yes; then
   BUILD_LDFLAGS=""
   BUILD_EXEEXT=""
 else
   BUILD_CC='$(CC)'
   BUILD_CFLAGS='$(CFLAGS)'
   BUILD_CXX='$(CXX)'
   BUILD_CXXFLAGS='$(CXXFLAGS)'
   BUILD_LDFLAGS='$(LDFLAGS)'
-  dnl ## 2012/07/31: Commented out special build requirements
-  dnl ## for Sun compiler now that gendoc.cc is no longer part of build.
-  dnl ##################################################################
-  dnl #case $canonical_host_type in
-  dnl #  sparc-sun-solaris2*)
-  dnl #    if test "$GCC" != yes; then
-  dnl #      ## The Sun C++ compiler never seems to complete compiling
-  dnl #      ## gendoc.cc unless we reduce the optimization level...
-  dnl #      ## BUILD_CXXFLAGS="-g -O1"
-  dnl #    fi
-  dnl #  ;;
-  dnl #esac
   BUILD_EXEEXT='$(EXEEXT)'
 fi
 
 AC_ARG_VAR([BUILD_CC],
   [build system C compiler (used if cross compiling)])
 AC_ARG_VAR([BUILD_CFLAGS],
   [build system C compiler flags (used if cross compiling)])
 AC_ARG_VAR([BUILD_CXX],
